From a9dda4dabe288258c6f82b84ce2e2b56289bfefd Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 19 Mar 2020 14:19:47 +0200
Subject: [PATCH] [SW-10047] pll: remove hl_pll_bypass_get()

The hl_pll_bypass_get() function was in use by the Gaudi pre-MF W/A
implementation. After the W/A code was removed, the function is not
used anymore.
Remove the hl_pll_bypass_get() function.

Change-Id: I60e2e6a17bd2d52ab010e55138574049d7a32768
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index a82c2c6..610a1f1 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -18,13 +18,6 @@
 	cycles_per_usec = hl_pll_get_cpu_cycles_per_usec();
 }
 
-enum hl_pll_bypass_val hl_pll_bypass_get(u32 pll_base, enum hl_pll_div_out out)
-{
-	int offset = out * sizeof(u32);
-
-	return regmap_readl(pll_base + HL_PLL_DIV_SEL + offset);
-}
-
 void hl_pll_udelay(unsigned int usec)
 {
 	u64 cnt = (cycles_per_usec * usec) / MIN_CYCLES_PER_ITER;
diff --git a/common/pll.h b/common/pll.h
index d53f382..33dd8b4 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -59,8 +59,6 @@
 	int	div_sel[HL_PLL_DIV_SIZE];
 };
 
-enum hl_pll_bypass_val hl_pll_bypass_get(u32 pll_base,
-					 enum hl_pll_div_out out);
 void hl_pll_init(void);
 int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg);
 u64 hl_pll_get_cpu_cycles_per_usec(void);
